charts/opencloud/README.md: add global.domain.wopi

kastl-ars opened the pull request:
fix #54

michaelstingl reviewed (changes requested):
Good catch on the missing documentation! However, the default value should be `wopiserver.opencloud.test` (not `collaboration.opencloud.test`) to match:

1. The actual value in `values.yaml` line 35:
   ```yaml
   wopi: wopiserver.opencloud.test
   ```

2. The opencloud-compose repository which also uses:
   ```
   WOPISERVER_DOMAIN=wopiserver.opencloud.test
   ```

Could you please update the PR to show:
```
| `global.domain.wopi` | Domain for WOPI server | `wopiserver.opencloud.test` |
```

This ensures consistency across all deployment methods.

kastl-ars commented:
Thanks for the review, @michaelstingl. I changed the domain as you suggested and removed the unused `collaboration.wopiDomain` parameter (at least it is unused in the templates in `charts/opencloud/templates/`

michaelstingl reviewed (approved):
LGTM! 👍 

The documentation now correctly reflects the actual helm chart implementation:
- ✅ Added missing \ parameter
- ✅ Removed non-existent \ 
- ✅ Updated HTTPRoute documentation
- ✅ Helm lint passes
- ✅ Template rendering uses correct domain

Thanks for fixing this documentation issue!
